### PR TITLE
:sparkles: support confirm control and manual tasks

### PR DESCRIPTION
**Pull request by roblen45**
**Changes:**

1. :sparkles: support confirm control
2. :sparkles: support manual tasks

PR: #9 
---

See the [examples](#example) for inspiration.

## How can others test the changes?

> Describe how others can test your changes (you can remove this section for typo fixes etc.)

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

### CODE REVIEW

File: src/components/components.d.ts [hunk 1]
```diff
@@ -8,17 +8,30 @@
 import '@stencil/core';
 
 
+import {
```
Comment (Vyperus) [inline src/components/components.d.ts]: For one Import i like this more:

`import {IManualTask} from './manualtask-component/imanualtask';`

File: src/components/components.d.ts [hunk 2]
```diff
@@ -8,17 +8,30 @@
...
+  IManualTask,
+} from './manualtask-component/imanualtask';
 import {
```
Comment (Vyperus) [inline src/components/components.d.ts]: `import {IUserTask} from './usertask-component';`

File: src/components/components.d.ts [hunk 3]
```diff
@@ -8,17 +8,30 @@
...
   IUserTask,
-} from './usertask-component/iusertask';
+} from './usertask-component';
 
 
 export namespace Components {
 
+  interface ManualtaskComponent {
+    'manualtask': IManualTask;
+  }
```
Comment (Vyperus) [inline src/components/components.d.ts]:
```suggestion
  }
  
```

File: src/components/components.d.ts [hunk 4]
```diff
@@ -64,6 +79,12 @@ declare global {
   }
```
Comment (Vyperus) [inline src/components/components.d.ts]:
```suggestion
```

File: src/components/components.d.ts [hunk 5]
```diff
@@ -64,6 +79,12 @@ declare global {
...
 
 
+  interface HTMLManualtaskComponentElement extends Components.ManualtaskComponent, HTMLStencilElement {}
```
Comment (Vyperus) [inline src/components/components.d.ts]:
```suggestion
  interface HTMLManualtaskComponentElement extends Components.ManualtaskComponent, HTMLStencilElement {}
  
```

File: src/components/components.d.ts [hunk 6]
```diff
@@ -64,6 +79,12 @@ declare global {
...
+  var HTMLManualtaskComponentElement: {
+    prototype: HTMLManualtaskComponentElement;
+    new (): HTMLManualtaskComponentElement;
+  };
+
   interface HTMLDynamicUsertaskComponentElement extends Components.DynamicUsertaskComponent, HTMLStencilElement {}
```
Comment (Vyperus) [inline src/components/components.d.ts]:
```suggestion
  interface HTMLDynamicUsertaskComponentElement extends Components.DynamicUsertaskComponent, HTMLStencilElement {}
  
```

File: src/components/components.d.ts [hunk 7]
```diff
@@ -8,17 +8,30 @@
 import '@stencil/core';
```
Comment (Vyperus) [inline src/components/components.d.ts]: Oh i just saw that this file may be auto generated, if so ignore my comments on this file.

Maybe think about a possibility to ignore this on github.

File: src/components/manualtask-component/imanualtask.ts [hunk 8]
```diff
@@ -0,0 +1,9 @@
+export interface IManualTask {
+  id: string;
+  flowNodeInstanceId?: string;
```
Comment (Vyperus) [inline src/components/manualtask-component/imanualtask.ts]: I would like to sort interfaces a little bit like this:

1) All members that are necessary.
2) Everything optional.

File: src/components/manualtask-component/manualtask-component.tsx [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+  shadow: false,
+})
+export class ManualTaskComponent {
+  @Prop() public manualtask: IManualTask;
+  @Event() public continued: EventEmitter;
+  @Event() public canceled: EventEmitter;
+
+  public render(): any {
+    const hasManualTask: boolean = this.manualtask !== undefined && this.manualtask !== null;
+    if (hasManualTask) {
```
Comment (Vyperus) [inline src/components/manualtask-component/manualtask-component.tsx]:
```suggestion
    if (hasManualTask) {
    
```

File: src/components/manualtask-component/manualtask-component.tsx [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+          <h3 class='card-title'>{this.manualtask.name}</h3>
+          <br></br>
+          <div class='float-right'>
+            <button type='button' class='btn btn-secondary' onClick={(e: Event): void => this._handleCancel(e)}
+              id='dynamic-ui-wrapper-cancel-button'>Cancel</button>&nbsp;
+          <button type='button' class='btn btn-primary' onClick={(e: Event): void => this._handleContinue(e)}
+              id='dynamic-ui-wrapper-continue-button'>Continue</button>
+          </div>
+        </div>
+      </div>;
```
Comment (Vyperus) [inline src/components/manualtask-component/manualtask-component.tsx]:
```suggestion
      </div>;
      
```

File: src/components/manualtask-component/manualtask-component.tsx [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    } else {
```
Comment (Vyperus) [inline src/components/manualtask-component/manualtask-component.tsx]:
```suggestion
    } else {
    
```

File: src/components/usertask-component/dynamic-usertask-component.tsx [hunk 12]
```diff
@@ -58,30 +59,67 @@ export class DynamicUserTaskComponent {
...
-            {
-              this._formFields.map((formField: any) => {
-                return formField.render();
-              })
-            }
-            <input type='submit' class='btn btn-primary' id='dynamic-ui-wrapper-continue-button' value='Abschließen'></input>
-          </form>
-        </div>
-      </div>;
+      if (isConfirmUserTask) {
```
Comment (Vyperus) [inline src/components/usertask-component/dynamic-usertask-component.tsx]:
```suggestion
      if (isConfirmUserTask) {
      
```

File: src/components/usertask-component/dynamic-usertask-component.tsx [hunk 13]
```diff
@@ -58,30 +59,67 @@ export class DynamicUserTaskComponent {
...
+        return this._renderConfirmUserTask();
+      } else {
```
Comment (Vyperus) [inline src/components/usertask-component/dynamic-usertask-component.tsx]:
```suggestion
      } else {
      
```

File: src/components/usertask-component/dynamic-usertask-component.tsx [hunk 14]
```diff
@@ -58,30 +59,67 @@ export class DynamicUserTaskComponent {
...
+        return this._renderUserTask();
+      }
     } else {
```
Comment (Vyperus) [inline src/components/usertask-component/dynamic-usertask-component.tsx]:
```suggestion
    } else {
    
```

File: src/components/usertask-component/dynamic-usertask-component.tsx [hunk 15]
```diff
@@ -58,30 +59,67 @@ export class DynamicUserTaskComponent {
...
+          <button type='button' class='btn btn-danger' onClick={(e: Event): void => this._handleDecline(e)}
+            id='dynamic-ui-wrapper-decline-button'>Decline</button>&nbsp;
+          <button type='button' class='btn btn-success' onClick={(e: Event): void => this._handleProceed(e)}
+            id='dynamic-ui-wrapper-proceed-button'>Proceed</button>
+        </div>
+      </div>
+    </div>;
+  }
+
+  private _renderUserTask(): any {
```
Comment (Vyperus) [inline src/components/usertask-component/dynamic-usertask-component.tsx]:
```suggestion
  private _renderUserTask(): any {
  
```

File: src/components/components.d.ts (hunk 7 above)
Comment (roblen45) [reply src/components/components.d.ts]: you are wright

**Pull request merged**